.github/workflows/test-all.yml

```diff
@@ -90,7 +90,7 @@ jobs:
     - name: Set start timestamp env var
       run: echo "FIREBASE_CI_TEST_START_TIME=$(date +%s)" >> $GITHUB_ENV
     - name: Run unit tests
-      # Ignore auth and firestore since they're handled in their own separate jobs.
+      # Ignore auth data-connect and firestore since they're handled in their own separate jobs.
```

Comment:
nit: put a comma after "auth" and "data-connect" in `auth data-connect`

Reply:
Comment removed